build/install-build-tools.sh

```diff
@@ -35,7 +35,7 @@ if [[ -z $(grep `hostname` /etc/hosts) ]]; then
     exit -1
 fi
 
-if [[ "${DISTRO}" == "rhel" ]]; then
+if [ "x${DISTRO}" == "xrhel" -o "x$DISTRO" == "xcentos" ]; then
```

Comment:
No need to prompt for RHEL subscription with centos, surely that's the whole point :-)